node/nodejs_intro/README.md

```diff
@@ -1,65 +1,98 @@
 # Node.js Intro + The Event Loop
 
-# Topics
-- What is Node.js?
-- Diving deeper into Programming Languages
-- Google's Javascript Engine: V8
-- Know and understand the event loop.
-- Know the different phases of the event loop.
-
-
-# 1. What is Node.js?
-
-### Why not just call it Javascript for Server?
+# Objectives
+- `Understand the need for higher-level programming languages`
+- `Articulate how Javascript and Node.js are related, but different`
+- `Demonstrate the different phases of the event loop`
+
+## Key Vocabulary
+- Higher-level programming
+- Compiled vs Interpreted code
+- Single-threaded
+- Asynchronous
+- Event loop
+- Call stack
+- WebAPI
+
+# What is Node.js?
 
 If we go to the Node.js (Node, for short) [website](https://node.js.org/en/) we get the following definition:
 
 ```
 Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.
 ```
 
-At first glance that is a very confusing and technical description of what Node.js really is. This lesson we will go through these technical concepts and finally run our first Node.js code!
+That is a very technical, somewhat mysterious description. In this lesson we will clarify exactly what this means, understand how Node.js is different from the browser-based Javascript we've been working with so far, and finally we will run our first Node.js code!
 
-We will also understand why it's different from browser based Javascript we've been working with so far.
+In order to get there, we first need to take a step all the way back and look at some of the basics of how computer programs run.
 
-# 2. Diving deeper into Programming Languages
+# Objective 1: Higher-Level Programming Languages
 
 ### How does a computer program work?
 
 ![Chip](assets/microprocessor.jpg)
 
-In all our computers, whether they be desktop computers, laptops, or mobile phones, are these very smartly designed microprocessor chips. These are the brains behind your computer and they are the ones that do all the calculations, interpretations and logic for your computer.
+In all our computers, whether desktop computers, laptops, or mobile phones, there are very smartly designed microprocessor chips that act as the brains behind the computer. These microprocessor chips are the ones that do all the calculations, interpretations and logic for your computer, and they only understand `binary code` (0s and 1s).
```

Comment:
this might be a good place to explicitly state that these are all different environments

Reply:
👍